php/class-fm-bylines.php

```diff
@@ -558,6 +562,31 @@ public function get_byline_posts( $byline_id, $type = 'author', $params = array(
 		}
 
 		/**
+		 * Get WP Query for the posts for a single byline.
+		 * Use to create loops on single byline pages.
+		 * @param int. $byline_id
+		 * @param string. $byline_type
+		 * @param array. $params
+		 * @return object. WP_Query object.
+		 */
+		public function get_byline_posts_query( $byline_id, $type, $params = array() ) {
+			$byline_id = ( empty( $byline_id ) ) ? get_the_ID() : $byline_id;
+			$type = ( empty( $type ) ) ? $this->get_byline_type() : $type;
```

Comment:
As noted above, I don't require it here so it can guess from the byline page. Not sure what the best way to proceed is here then.